lib/pkg/flow-base/lib/FlowHelpers.js

```diff
@@ -144,12 +144,32 @@ async function canFindFlow(flowPath: string): Promise<boolean> {
 }
 
 /**
- * @return The path to Flow on the user's machine. It is recommended not to cache the result of this
- *   function in case the user updates his or her preferences in Atom, in which case the return
- *   value will be stale.
+ * @return The path to Flow on the user's machine. First using the the user's config, then looking into
+ *   the node_modules for the project.
+ *   It is recommended not to cache the result of this function in case the user updates his or her 
+ *   preferences in Atom, in which case the return value will be stale.
  */
-function getPathToFlow(): string {
-  return global.vscode.workspace.getConfiguration('flow').get('pathToFlow')
+async function getPathToFlow(): Promise<string> {
+  const config = global.vscode.workspace.getConfiguration('flow')
+  const userPath = config.get('pathToFlow')
```

Comment:
Does this default to `flow` if not explicitly set? Does this mean that if `flow` is in the path, then the `useNPMPackagedFlow` option is ignored?